yb-voyager/cmd/exportData.go

```diff
@@ -385,7 +385,7 @@ func extractTableListFromString(fullTableList []*sqlname.SourceName, flagTableLi
 		utils.PrintAndLog("Unknown table names %v in the %s list", unknownTableNames, listName)
 		utils.ErrExit("Valid table names are %v", fullTableList)
 	}
-	return result
+	return sqlname.Uniq(result)
```

Comment:
checkout if `lo.Uniq()` can be used here
refer - [link](https://github.com/samber/lo#:~:text=names%20%3A%3D%20lo.Uniq%5Bstring%5D(%5B%5Dstring%7B%22Samuel%22%2C%20%22John%22%2C%20%22Samuel%22%7D))

Reply:
yes, removed the utils.Uniq() as well from import side.